vignettes/rgl.Rmd

```diff
@@ -230,6 +230,37 @@ provide low-level support for `r linkfn("setAxisCallbacks")`.
 
 ## Controlling the Look of the Scene
 
+### Camera angle
+
+By default when you open a new plot:
+
+* The x axis goes from left to right.
+* The y axis goes from near the camera to far away.
+* The z axis goes from down to up.
+
+You can change the camera angle simply by dragging the picture with
+the mouse.
+
+To set the camera angle programmatically, use `r indexfns("view3d")`. 
+This uses polar coordinates:
+
+* `theta` sets the rotation around the z axis, in degrees. 
```

Comment:
This is just one way to set the camera angle, and there's ambiguity about what "the z axis" means, because it might have already been rotated.  

I would recommend setting the angle using the mouse, then saving the `par3d("userMatrix")` value.  That same orientation can then be restored using `par3d(userMatrix = savedMatrix)`.  But if you want to use `view3d()`, it might be less ambiguous to say "`theta` sets the rotation about the vertical axis" and "`phi` sets the rotation about the horizontal axis".